docker/r10k/Dockerfile

```diff
@@ -31,6 +31,10 @@ LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
       org.label-schema.schema-version="1.0" \
       org.label-schema.dockerfile="/Dockerfile"
 
+# Add a puppet user to run r10k as for consistency with puppetserver
+# The alpine image uses the 'ping' group with gid 999, so use that instead of
+# a 'puppet' group
+RUN adduser puppet -D -G ping -u 999
```

Comment:
I have no idea if Alpine has any guarantees around group id to name mappings staying consistent?

Since we really care about the numeric value in the end, should we just use numeric values everywhere?

Reply:
ah, good call. we should probably do something safer here where we see if there's a `999` group (apparently that wasn't in 3.3 for example), add it if it doesn't exist, and add the user to  whatever group 999 is if it does exist

Reply:
@Iristyle updated!